service/test_file_transaction_service.py

```diff
@@ -40,10 +41,40 @@ def test_GetStatus(state_input, azure_input, expected_output, desc):
     s = server.Server()
     s.set_state(state_input)
     file_trans_svc = FileTransactionService(s)
-    utility.block_blob_service = azure_input
+    azure_client.block_blob_service = azure_input
     req = FileTransactionRequest()
     ctx = FakeContext()
 
     response = file_trans_svc.GetStatus(req, ctx)
     assert response.code == expected_output["code"]
     assert response.message == expected_output["message"]
+
+
+@pytest.mark.parametrize("uuid_input, expected_output, desc",
+                         [
+                             ("1234abcd5454efef8842ll3fsc",
+                              {"code": grpc.StatusCode.UNKNOWN.value[0],
+                               "message": "user folder already exists"},
```

Comment:
I don't understand why the first case means that the folder already exists. At this point Azurite should not have any folders. Am I missing something?

Reply:
It only exists because I put it in there intentionally to test that case.

Reply:
This will break if we run this in the CI/CD pipeline. 
Consider doing the following:
- generate a valid **global random** uuid called `valid_uuid_1`
- first test case uses `valid_uuid_1` thus passing the test
- second test case uses `valid_uuid_1` thus failing the test
- more test cases...

Reply:
Note that the uuid is randomly generated.